programs/smart-wallet/src/state.rs

```diff
@@ -200,8 +200,23 @@ pub struct SubaccountInfo {
 pub struct InstructionBuffer {
     /// Execution count on this buffer.
     pub exec_count: u8,
-    /// Key that can write to the buffer.
+    /// Sequence of the ownership set.
+    ///
+    /// This may be used to see if the owners on the multisig have changed
+    pub owner_set_seqno: u32,
+    /// - If set to [crate::NO_ETA], the instructions in the [InstructionBuffer::instructions] may be executed at any time.
+    /// - Otherwise, instructions may be executed at any point after the ETA has elapsed.
+    pub eta: i64,
+    /// Time denoting when buffer is ready to be executed.
+    pub finalized_at: i64,
```

Comment:
Delete this, and replace the `finalized` check with checking if `writer` is `system_program::ID`.